### PR TITLE
cleanup script formatting 

### PR DESCRIPTION
**Pull request by CvH**
That is an first attempt to clean-up our bash scripts and bring them to some common format.
The needed changes to add `" "` around variables and convert `{ }` to a proper usage are too many changes for now (around 2700 additional changes).

The current style follows the standard style guide from here https://google.github.io/styleguide/shellguide.html .
I tried to not include any functional change.

That PR is mainly focus to:
- indents
- remove useless spaces
- proper spaces around pipes etc
- case statement wrappings 
- converting legacy bash to current format ( ` 'pwd' ` -> `$(pwd)` 
- redirect operators will not followed by a space (we had a 50:50 mix with and without spaces)

Things where I'm not sure if that's better now even now its properly formatted.

https://github.com/LibreELEC/LibreELEC.tv/compare/master...CvH:LibreELEC.tv:13.0/shfmt?expand=1#diff-ba6182c0ace62ebedbca279e78875580f33ec80c17f80e562dcf7cabe2495fd1L150-L160

The bulk work was done by that script, additional I dropped some questionable changes that might introduce functional changes and optimized some stuff manually.

```
#!/bin/bash
find config/ packages/ projects/ scripts/ tools/ -type f -print0 | while read -r -d '' file_name; do
  if $(head "$file_name" | grep -q '#!/bin/bash\|#!/bin/sh'); then
    echo "$file_name"
    ./shfmt_v3.8.0_linux_amd64 -i 2 -ci -w "$file_name"
  fi
done
```

### CODE REVIEW

File: packages/addons/service/syncthing/source/bin/syncthing-service [hunk 1]
```diff
@@ -6,10 +6,9 @@
...
 
-STNODEFAULTFOLDER="1" syncthing -home=$ADDON_HOME           \
-                                -gui-address="$gui_address" \
-                                -logflags=0                 \
-                                -no-browser                 \
-                                -no-restart                 \
-                                &
-
+STNODEFAULTFOLDER="1" syncthing -home=$ADDON_HOME \
+  -gui-address="$gui_address" \
```
Comment (chewitt) [inline packages/addons/service/syncthing/source/bin/syncthing-service:10]: pershaps drop `-home` onto a line below to keep the params aligned

File: scripts/build [hunk 2]
```diff
@@ -171,7 +171,6 @@ if [ "${BUILD_WITH_DEBUG}" != "yes" ] && flag_enabled "strip" "yes"; then
   TARGET_MESON_OPTS+=" -Dstrip=true"
 fi
 
-
```
Comment (antonlacon) [inline scripts/build:174]: Double line breaks are done for grouping/dividing. In this example, TARGET from HOST options.

Comment (CvH) [reply scripts/build:174]: > Double line breaks are done for grouping/dividing.

I really do not like the idea of grouping with wrong style formats. I guess I add a comment instead ?

Comment (antonlacon) [reply scripts/build:174]: I don't see how it's a wrong style. What rule is the script enforcing and can it be disabled? Can it show just changes for applying that rule? The spot I commented is just an example and this double line break is done in other places.

Comment (CvH) [reply scripts/build:174]: @antonlacon looked through all changes and we have just 3 cases where this is changed and in all cases the double line breaks are followed by a comment that not really separates anything 
from my perspective that is a non issue as the extra line break makes nothing more readable or similar
image

image

**Review by heitbaum — APPROVED** [review]

**Pull request merged**